examples/apps/likes-and-comments/src/fluidObject.tsx

```diff
@@ -23,16 +24,17 @@ const defaultImgUrl = "https://picsum.photos/id/221/1200/800";
 export class LikesAndComments extends SyncedDataObject {
     public static get Name() { return "LikesAndComments"; }
 
-    public static readonly factory = new DataObjectFactory(
-        LikesAndComments.name,
-        LikesAndComments,
-        [
-            SharedCounter.getFactory(),
-            SharedObjectSequence.getFactory(),
-            SharedString.getFactory(),
-        ],
-        {},
-    );
+    public static readonly factory =
+        new DataObjectFactory<LikesAndComments, unknown, unknown, IEvent>(
```

Comment:
should probably be `<LikesAndComments, object, unknown, IEvent>` like the others?